### PR TITLE
Added vibration to bridge

### PR DESCRIPTION
**Pull request by Marcosld**

### CODE REVIEW

File: src/__tests__/device-test.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ import {
     requestDeviceImei,
     internalNavigation,
     dismiss,
+    requestVibration,
```
Comment (pladaria) [inline src/__tests__/device-test.ts]: I'd name this `vibrate`

File: src/device.ts [hunk 2]
```diff
@@ -34,3 +34,13 @@ export const dismiss = (onCompletionUrl?: string): Promise<void> =>
             onCompletionUrl,
         },
     });
+
+type VibrationsAvailable = 'error' | 'success';
```
Comment (pladaria) [inline src/device.ts]: `type VibrationType = 'error' | 'success';`

but you could simply define that inline

Comment (Marcosld) [reply src/device.ts]: oops, true

File: src/__tests__/device-test.ts (hunk 1 above)
Comment (Marcosld) [reply src/__tests__/device-test.ts]: Left it as requestVibration as some devices won't do anything and we won't receive a confirmation.

**Pull request merged**